contracts/CarryPublicTokenCrowdsale.sol

```diff
@@ -174,14 +187,23 @@ contract CarryPublicTokenCrowdsale is CappedCrowdsale, Pausable {
         );
 
         // See also the comment on the individualMaxCaps above.
-        uint256 latestTimestamp = 0;
         uint256 individualMaxWei = 0;
-        for (uint256 i = 0; i < individualMaxCaps.length; i++) {
+        for (uint i = 0; i < individualMaxCaps.length; i++) {
+            uint256 capTimestamp = individualMaxCaps[i].timestamp;
             // solium-disable-next-line security/no-block-members
-            if (individualMaxCaps[i].timestamp <= block.timestamp &&
-                individualMaxCaps[i].timestamp > latestTimestamp) {
-                latestTimestamp = individualMaxCaps[i].timestamp;
+            if (capTimestamp <= block.timestamp) {
```

Comment:
Can we use inversed `individualMaxCaps` ?  

e.g.
```js
var cap = (now, caps) => {
    for(c of caps) {
        if (now >= c.time) {
            return c.cap;
        }
    }
}

console.log('at 2: ' + cap(2, [{time: 5, cap:5}, {time: 3, cap:3}, {time: 1, cap:1}]));
console.log('at 3: ' + cap(3, [{time: 5, cap:5}, {time: 3, cap:3}, {time: 1, cap:1}]));
console.log('at 7: ' + cap(7, [{time: 5, cap:5}, {time: 3, cap:3}, {time: 1, cap:1}]));
```

Reply:
Does that reversed table (or iterating in descending order) purpose to replace the gas optimization below?

Reply:
Yes, I intended early `break` before read whole list. but first of investor's should read whole list yet. 😭